config/cryptominisat.m4

```diff
@@ -20,13 +20,22 @@ elif test -n "$with_cryptominisat"; then
       [--with-cryptominisat-dir=PATH],
       [path to top level of cryptominisat source tree]
     ),
-    [CRYPTOMINISAT_HOME="$withval"],
-    [ if test -z "$CRYPTOMINISAT_HOME"; then
+    CRYPTOMINISAT_HOME="$withval",
+    [ if test -z "$CRYPTOMINISAT_HOME" && ! test -e "$ac_abs_confdir/cryptominisat4/install/bin/cryptominisat"; then
         AC_MSG_FAILURE([must give --with-cryptominisat-dir=PATH or define environment variable CRYPTOMINISAT_HOME!])
```

Comment:
Nit: Can we update this error message to say that using the `get-cryptominisat4` script is an option, too?